lib/rubocop/cop/sorbet/signatures/enforce_signatures.rb

```diff
@@ -119,6 +180,74 @@ def return_type_placeholder
           cop_config["ReturnTypePlaceholder"] || "T.untyped"
         end
 
+        def allow_rbs?
+          cop_config["AllowRBS"] == true
+        end
+
+        def signature_style
+          config_value = cop_config["Style"]
+          return config_value if config_value
```

Comment:
Should we raise if the option passed in incorrect?

I can imagine someone using `Style: RBS` and getting confused by the result.